features/api/metrics/update.feature

```diff
@@ -0,0 +1,39 @@
+@javascript
+Feature: Product > Integration > Metrics index > Metric
+  In order to change my metrics for whatever reason
+  As a provider
+  I want to be able to modify them
+
+  Background:
+    Given a provider is logged in
+    And the following metrics:
+      | Antipasti |
+    And the following methods:
+      | Burrata |
+    And I go to the metrics and methods page
+
+  Scenario: Edit a method from the index page
+    Given I change to tab "Methods"
+    When I follow "Burrata"
+    And I fill in "Friendly name" with "Burrata Panzanella"
+    And I press "Update Method"
+    Then I should see "Burrata Panzanella"
+    And method "Burrata Panzanella" should have the following:
+      | Friendly name | Burrata Panzanella |
+      | Unit          | hit                |
+
+  Scenario: Can't change system name of default metric
+    Given I change to tab "Metrics"
+    When I follow "Hits"
+    Then I should not see field "metric[system_name]"
```

Comment:
This is new, right? Because right now the field is there; it's just disabled, yet still visible.

Reply:
Probably use a step where a hidden field is expected, if it is the case

Reply:
Yeah good catch

Reply:
Yes, for capybara to "not see" equals "see hidden or disabled". But there is this other step `see field X disabled` which is more specific.